pkgs/top-level/all-packages.nix

```diff
@@ -5902,6 +5902,8 @@ in
 
   nwdiag = with python3Packages; toPythonApplication nwdiag;
 
+  nxdomain = python3.pkgs.callPackage ../tools/networking/nxdomain { };
```

Comment:
Ok, now that you have a binary, I think the typical pattern is to do `nxdomain = with python3Packages; toPythonApplication nxdomain;` and call the nxdomain from `pkgs/top-level/python-packages.nix`

Reply:
The PR was merged before I could apply your feedback. I'll do that the next time I need to update the package. Thanks.